### PR TITLE
create translation for Dutch (Netherlands)

### PR DESCRIPTION
**Pull request by Joppla**

### CODE REVIEW

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<resources>
+    <string name="app_name">Android versie</string>
+    <string name="codename">Code name: %s\n</string>
+    <string name="version">Versie: %s (API %d)\n</string>
+    <string name="based_on">ROM gebaseerd op: %s\n</string>
+    <string name="preview">Preview-versie van %s (API %d)\n</string>
+    <string name="patch">Patch niveau: %s\n</string>
+    <string name="made_by">Auteur</string>
+    <string name="license">Licentie</string>
+    <string name="android_trademark">Android is a trademark of Google LLC.</string>
```
Comment (mueller-ma) [inline app/src/main/res/values-nl/strings.xml:10]: This string doesn't seem to be translated. Can you translate or remove it?

Comment (Joppla) [reply app/src/main/res/values-nl/strings.xml]: Maybe some words are the same as in English and it looks not translated, but is is. Example: License in English, Licentie in Dutch ...

Comment (mueller-ma) [reply app/src/main/res/values-nl/strings.xml]: I was only commenting on `android_trademark`, which is translated now. Thanks!

Comment (Joppla) [reply app/src/main/res/values-nl/strings.xml]: Oke, You are welcome

**Pull request merged**